pages/apply.html

```diff
@@ -152,26 +165,27 @@
         <div class="usa-input">
 
           <input type="checkbox" name="entry.1263390573" value="Verify your ability to work full-time" id="group_1263390573_1" role="checkbox" class="ss-q-checkbox">
-          <label for="group_1263390573_1">Verify your ability to work full-time</label>
+          <label for="group_1263390573_1">I am able to work full time (40 hours/week).</label>
+          <label for="group_1263390573_2">I am not able to work full time.</label>
```

Comment:
Separately, in case you're curious: the display issue was due to not encapsulating these in a list (`<ul>` and `<li>`).